lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss

```diff
@@ -1,204 +1,57 @@
-.adf {
-    &-textitem-edit-icon.mat-icon {
-        font-size: var(--theme-subheading-2-font-size);
-        width: 16px;
-        height: 16px;
-        color: var(--adf-theme-foreground-text-color-025);
+.adf-card-view-textitem {
+    .adf-textitem-error {
+        font-size: var(--theme-caption-font-size);
+        padding-top: 6px;
+
+        ul {
+            margin: 0;
+            padding: 0;
+            list-style-type: none;
+
+            li {
+                margin: 0;
+                padding: 0;
+            }
+        }
     }
 
-    &-textitem-action.mat-icon-button {
+    .adf-textitem-action {
         width: 20px;
         height: 20px;
         line-height: 20px;
         color: var(--adf-theme-foreground-text-color-025);
-    }
-
-    &-textitem-action:hover,
-    &-textitem-action:focus {
-        color: var(--adf-theme-foreground-text-color);
-    }
-
-    &-update-icon {
-        padding-left: 13px;
-    }
-
-    &-textitem-readonly {
-        cursor: pointer !important;
 
-        &:hover .adf-textitem-action,
-        &:focus .adf-textitem-action {
+        &:hover,
+        &:focus {
             color: var(--adf-theme-foreground-text-color);
         }
     }
 
-    &-textitem-chip-list-container {
-        margin-bottom: 25px !important;
-        margin-top: 6px;
-
+    .adf-textitem-chip-list-container {
         .mat-form-field-label {
             margin-top: 6px;
         }
     }
 
-    &-textitem-clickable {
+    .adf-textitem-clickable {
```

Comment:
should not be text-item instead of textitem?

Reply:
closing based on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379735582 comment